instrumentation/rxjava/rxjava-3.0/javaagent/src/test/groovy/RxJava3WithSpanInstrumentationTest.groovy

```diff
@@ -619,6 +737,40 @@ class RxJava3WithSpanInstrumentationTest extends AgentInstrumentationSpecificati
     }
   }
 
+  def "should capture span for eventually errored Flowable"() {
```

Comment:
```suggestion
  def "should capture span for eventually canceled Flowable"() {
```

Reply:
ugh, copy/paste fail.  I'm surprised it doesn't error when two methods have identical names like that.